pkgs/by-name/gr/gruvbox-plus-icons/package.nix

```diff
@@ -7,6 +7,7 @@
   gnome-icon-theme,
   hicolor-icon-theme,
   nix-update-script,
+  folder-color ? "plasma", # Supported colors: black blue caramel citron firebrick gold green grey highland jade lavender lime olive orange pistachio plasma pumpkin purple red rust sapphire tomato violet white yellow
```

Comment:
Nice, that's exactly what I was going to suggest. I think maybe eventually there could be a home-manager module as well for declaring the color, but this is great for immediate use.